config/ember-try.js

```diff
@@ -63,15 +63,34 @@ module.exports = async function () {
         },
       },
       {
-        name: 'ember-default-with-jquery',
-        env: {
-          EMBER_OPTIONAL_FEATURES: JSON.stringify({
-            'jquery-integration': true,
-          }),
+        name: 'ember-modifier@2',
+        npm: {
+          dependencies: {
+            'ember-modifier': '^2.1.2',
+          },
         },
+      },
+      {
+        name: 'ember-modifier@3.1',
         npm: {
-          devDependencies: {
-            '@ember/jquery': '^1.1.0',
+          dependencies: {
+            'ember-modifier': '^3.1.0',
+          },
+        },
+      },
+      {
+        name: 'ember-modifier@3.2',
+        npm: {
+          dependencies: {
+            'ember-modifier': '^3.2.7',
+          },
+        },
+      },
+      {
+        name: 'ember-modifier@^4.0.0-beta.0',
```

Comment:
@And012 let's rename this to `ember-modifier@^4.0.0-beta.1` as there was no beta0 published to npm (and don';t forget to update `.github/workflows/ci.yml`)